grading/load_config_json.cpp

```diff
@@ -108,41 +108,50 @@ void ArchiveValidatedFiles(nlohmann::json &whole_config) {
 
 void AddDockerConfiguration(nlohmann::json &whole_config) {
 
-  if (!whole_config["docker_enabled"].is_boolean()){
-    whole_config["docker_enabled"] = false;
+  //check if docker is globally enabled.
+  if (!whole_config["autograding_method"].is_string()) {
+    whole_config["autograding_method"] = "jailed sandbox";
```

Comment:
avoid spaces... switch to jailed_sandbox

Reply:
Fixed.